--- a/include/base/cef_atomicops.h	2015-12-07 13:28:31 +0300
+++ b/include/base/cef_atomicops.h	2015-12-16 11:25:41 +0300
@@ -177,7 +177,7 @@
 }  // namespace base
 
 // Include our platform specific implementation.
-#if defined(OS_WIN) && defined(COMPILER_MSVC) && defined(ARCH_CPU_X86_FAMILY)
+#if defined(OS_WIN) && defined(ARCH_CPU_X86_FAMILY)
 #include "include/base/internal/cef_atomicops_x86_msvc.h"
 #elif defined(OS_MACOSX)
 #include "include/base/internal/cef_atomicops_mac.h"
